### ISSUE TITLE
Batch Normalization error

### ISSUE THREAD

**Issue by zhichuande** [Microsoft/CNTK#138]
when Run the example from the Image/MNIST/Data folder using:
cntk configFile=../Config/01_OneHidden.cntk

EXCEPTION occurred: Undefined function or macro 'BatchNormalization' in BatchNormalization(t, sc, b, m, isd, eval = false, spatial = false, expAvgFactor = expAvg)

why? How to resovle it?
